frontend/src/lib/search-utils.ts

```diff
@@ -22,49 +22,38 @@ export const handleStandardComparison = (valueOne: string, valueTwo: string, ope
   }
 }
 
-export const handleSemverOperatorComparison = (
-  versionToCompare: string,
-  inputVersion: string,
-  operator: SearchOperator
-) => {
-  const coerceVersion = (version: string) => semver.coerce(version) ?? version
-  const coercedInputVersion = coerceVersion(inputVersion)
-  const coercedVersionToCompare = coerceVersion(versionToCompare)
-  const isValidSemver = (version: string | semver.SemVer) => semver.valid(version) !== null
+export const handleSemverOperatorComparison = (versionToCompare: string, input: string, operator: SearchOperator) => {
```

Comment:
Renamed `inputVersion` to `input` because this is just whatever the user entered, and that leaves me to use `inputVersion` to refer to the portion of that they entered that is a partial or full semver.